compiler/rustc_mir_build/src/build/mod.rs

```diff
@@ -570,7 +570,22 @@ fn should_abort_on_panic(tcx: TyCtxt<'_>, fn_def_id: LocalDefId, abi: Abi) -> bo
                 // Rust and `rust-call` functions are allowed to unwind, and should not abort.
                 Rust | RustCall => false,
                 // Other ABI's should abort.
-                _ => true,
+                Cdecl
+                | Fastcall
+                | Vectorcall
+                | Aapcs
+                | Win64
+                | SysV64
+                | PtxKernel
+                | Msp430Interrupt
+                | X86Interrupt
+                | AmdGpuKernel
+                | EfiApi
+                | AvrInterrupt
+                | AvrNonBlockingInterrupt
+                | RustIntrinsic
+                | PlatformIntrinsic
+                | Unadjusted => false,
```

Comment:
```suggestion
                | Unadjusted => true,
```
The latest commit changes `_ => true` to `/* lots of ABIs */ => false` and that seems unintended.

Reply:
:pray: _Thank_ you, that was very much unintended.